core/src/language/swift.rs

```diff
@@ -210,9 +210,9 @@ impl Language for Swift {
         let mut decs = self.get_default_decorators();
 
         // Check if this struct's decorators contains swift in the hashmap
-        if rs.decorators.contains_key("swift") {
+        if rs.decorators.contains_key(&SupportedLanguage::Swift) {
             // this unwrap is safe since we just checked if we contain the key
-            let swift_decs = rs.decorators.get("swift").unwrap();
+            let swift_decs = rs.decorators.get(&SupportedLanguage::Swift).unwrap();
```

Comment:
This could be improved by using `if let Some(...)`

Reply:
done.